VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
@WenMeng-NOAA - We need to decide what this version number should in develop branch.  See the discussion of this VERSION file in #140 conversation.

Reply:
@fossell The branch develop is the main development branch.  We would create release branch e.g. release/gfs_v16,
release/public_v1 from develop when specific application is in pre-implementation. Will we add version number from 
release branches and leave VERSION unfilled in branch develop?

Reply:
@WenMeng-NOAA That could work. I was debating whether we should leave VERSION out of develop completely, and only have it in specific application branches with corresponding version number for that application. I'm ok with either approach I think, though having an empty version file might be confusing to users/developers?  This is beyond this PR, but maybe I can try to create explanation of why we don't have a straightforward versioning system within develop.

Reply:
@fossell I think leave VERSION out of develop would be better. VERSION can be started from a application branch.

Reply:
@WenMeng-NOAA - Sounds good. I will delete VERSION file from this PR.

Reply:
@fossell deleting this file will result in a failed build.
Changes will be required in `CMakeLists.txt`.

Reply:
@aerorahul - The cmake build will fail, correct? Not other regular reg tests.  Can you then address this with appropriate mods to CMakeLists.txt when updating your #140 after we make this merge with the current release/public-v1?  This PR will be specifically for merging the current release/public-v1 as-is into develop, followed by any updates to your #140 PR to make updates to cmake accordingly.  Will that work?

Reply:
@fossell I cannot do that. My changes are built on top of what is in the public/release-v1.
A `PROJECT_VERSION` is required, so that downstream applications can "find" the package with a minimum version requirement.
The UFS depends on it.

Reply:
@fossell Can we use public release version number for VERSION? Right now, all other UPP versions for  ops. NCEP applications haven't use CMAKE  function yet.

Reply:
Per conversation with @WenMeng-NOAA and @aerorahul, decision was made to keep VERSION file in develop and add placeholder version number that will be changed in future after internal UPP decision is made on versioning. @aerorahul will also add comments in the VERSION file via hot fix merge to make it clear to developers and users that this is not official.